data/registry/tools-ruby-rspec-matcher.yml

```
@@ -0,0 +1,20 @@
+title: RSpec matchers
+registryType: utilities
+language: ruby
+tags:
+  - ruby
+  - rspec
+license: MIT
+description:
+  The RSpec matchers tool provides a set of matchers to make it easier to write
+  unit tests checking the presence of telemetry data.
+authors:
+  - name: Damien MATHIEU # cspell:disable-line
```

```suggestion
  - name: Damien Mathieu
```
@dmathieu as suggested move the cspell ignore to the top of the file. Also if you don't mind I would prefer to not uppercase your last name (except this is how you commonly write it, then ignore that suggestion)

---

I usually write it in full uppercase, to show which is my name and which is my last name. But not doing so is fine too.